atlas/templates/docker/Dockerfile.gotmpl

```diff
@@ -11,4 +11,4 @@ WORKDIR /bin
 COPY --from=builder /go/src/{{ .Root }}/{{ .Name }}/bin/server .
 {{ if .WithGateway }}COPY pkg/pb/*.swagger.json tmp/www/swagger{{ end }}
 {{ if .WithDatabase }}COPY --from=builder /go/src/{{ .Root }}/{{ .Name }}/db/migrations /db/migrations/{{end}}
-ENTRYPOINT ["server"{{ if .WithGateway}}, "--gateway.swaggerFile", "www/{{ .Name }}.swagger.json"{{ end }}]
+ENTRYPOINT ["server"{{ if .WithGateway}}, "--gateway.swaggerFile", "tmp/www/{{ .Name }}.swagger.json"{{ end }}]
```

Comment:
This is actually a good catch. But with this change it will not work as expected.
Line 12 will copy `*.swagger.json` to a file named `swagger` in `tmp/www` folder, while line 14 will contain something like `tmp/www/myapp.swagger.json` which do not exist

I think the right thing would be:
```
{{ if .WithGateway }}COPY pkg/pb/*.swagger.json www/swagger.json{{ end }}
...
ENTRYPOINT ["server"{{ if .WithGateway}}, "--gateway.swaggerFile", "www/swagger.json"{{ end }}]
```

Reply:
You're right. will make that change